c7n/resources/vpc.py

```diff
@@ -2185,6 +2185,78 @@ class resource_type(query.TypeInfo):
         taggable = False
 
 
+@KeyPair.filter_registry.register('unused')
+class UnusedKeyPairs(Filter):
+    """Filter for used or unused keys. The default is unused
+       but can be changed by using the state property.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
```

Comment:
fwiw, this is fine, but you only have to indent 2 past a code block for rst, this appears to be 8

Reply:
thanks for the suggestion, improved in f93f217